src/partials/ecosystem/hit.tsx

```diff
@@ -26,8 +27,8 @@ export default function Hit({ hit }: { hit: EcosystemElement }) {
       <div className="flex gap-2 flex-wrap mt-auto">
         <Pill color={TECHNOLOGY_COLOR_MAP[hit.technology]}>{hit.technology}</Pill>
         <Pill color={TYPE_COLOR_MAP[hit.type]}>{hit.type}</Pill>
-        <Pill color={""}>{hit.category}</Pill>
+        <Pill color={''}>{hit.category}</Pill>
```

Comment:
```suggestion
        <Pill >{hit.category}</Pill>
```

Can this be remove if it's empty?